app.py

```diff
@@ -52,11 +52,12 @@ def index():
         Notams = ParseNOTAM.ParseNOTAM(apiOutputs)
         endTime = time.time()    # Record end time
         print(f"time parsing: {endTime - startTime} seconds\n")
+        

```

Comment:
This was used to look at the loading animation without having to call an actual request of NOTAMs (wary of the limit we have even though it is a lot.)